### PR TITLE
RPC eth_signTransaction/eth_sign/eth_call endpoints

### PR DESCRIPTION
**Pull request by saibatizoku**
These endpoints make use of mocked account secrets for signatures of Ethereum transactions and data strings.

### CODE REVIEW

File: explorer/Cargo.toml [hunk 1]
```diff
@@ -41,16 +41,16 @@ http-zipkin = "0.3.0"
...
-chain-addr = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-core = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-crypto = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-impl-mockchain = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-time = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-vote = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-ser = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-chain-network = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
-imhamt = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "master"}
+cardano-legacy-address = {git = "https://github.com/input-output-hk/chain-libs.git", branch = "evm-crypto-signature"}
```
Comment (Mr-Leshiy) [inline explorer/Cargo.toml:44]: Need to switch back to the `master` branch

File: jormungandr/src/jrpc/eth_transaction/logic.rs [hunk 2]
```diff
@@ -137,12 +145,38 @@ pub async fn estimate_gas(tx: Transaction, context: &Context) -> Result<Number,
...
+        sig
+    };
+    Ok(H512::from_slice(&signature[..]))
 }
 
-pub fn call(_tx: Transaction, _number: BlockNumber, _context: &Context) -> Result<Bytes, Error> {
-    // TODO implement
-    Ok(Default::default())
+pub async fn call(tx: Transaction, _: BlockNumber, context: &Context) -> Result<Bytes, Error> {
+    let account_secret = context
```
Comment (Mr-Leshiy) [inline jormungandr/src/jrpc/eth_transaction/logic.rs:167]: You can remove all these parts related to signing and replace it with the following
```
    let blockchain_tip = context.blockchain_tip()?.get_ref().await;
     Ok(blockchain_tip
         .ledger()
         .call_evm_transaction(tx.into())
         .map_err(Box::new)?
         .into())
```

Comment (saibatizoku) [reply jormungandr/src/jrpc/eth_transaction/logic.rs]: Done. Thank you.

File: explorer/Cargo.toml (hunk 1 above)
Comment (saibatizoku) [reply explorer/Cargo.toml]: Done, now that the dev branch is in master, I've reset this and updated Cargo.lock

**Pull request merged**